control/lane_departure_checker/include/lane_departure_checker/lane_departure_checker.hpp

```diff
@@ -78,13 +78,19 @@ struct Input
   std::vector<std::string> boundary_types_to_detect{};
 };
 
+struct DerivativeDeviation
+{
+  double longitudinal_vel{0.0};
+};
+
 struct Output
 {
   std::map<std::string, double> processing_time_map{};
   bool will_leave_lane{};
   bool is_out_of_lane{};
   bool will_cross_boundary{};
   PoseDeviation trajectory_deviation{};
+  DerivativeDeviation trajectory_der_deviation{};
```

Comment:
`der` is hard to understand for developers. Please do not abbreviate the word too much.

Reply:
fixed also in https://github.com/autowarefoundation/autoware.universe/commit/821cf29de7f95895292c453da6570cf537339121